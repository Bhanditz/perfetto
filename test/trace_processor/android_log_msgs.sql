create view v1 as select tag, count(*) from android_logs group by tag order by 2 desc limit 5

create view v2 as select tag, count(*) from android_logs group by tag order by 2 asc limit 5;

create view v3 as select tag, count(*) from android_logs where msg like '%wakelock%' group by tag;

select * from v1 union all
select '-----', 0 union all
select * from v2 union all
select '-----', 0 union all
select * from v3;