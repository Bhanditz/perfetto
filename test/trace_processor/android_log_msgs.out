


"tag","count(*)"
"mm-camera",562
"NxpTml",109
"sysui_multi_action",101
"NuPlayerDriver",60
"CAM_Profiler",57
"-----",0
"AlarmManager",1
"AudioTrack",1
"CAM_ActvCamDevTrckr",1
"CAM_AutoFlashZslImgFltr",1
"CAM_FlashBasedPhotoCmd",1
"-----",0
"GnssLocationProvider",8
"power_screen_broadcast_done",2
"power_screen_broadcast_send",2
"power_sleep_requested",1
